runtime/lua/vim/treesitter.lua

```diff
@@ -245,6 +249,64 @@ function M.get_captures_at_position(bufnr, row, col)
   return matches
 end
 
+---Gets a list of captures under the cursor
+---
+---@param winnr number|nil Window handle or 0 for current window (default)
+---
+---@returns (table) Named node under the cursor
+function M.get_captures_at_cursor(winnr)
```

Comment:
`getcurpos()` already returns the cursor position. Perhaps all of these `_at_position` functions should accept a table in that form. Then we can avoid endlessly ading `_at_cursor` variants to everything that takes a region.